go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.7.0
 	github.com/temporalio/ringpop-go v0.0.0-20200708034907-1e016ebb537a
-	github.com/temporalio/tctl v0.0.0-20210722235117-a400f112efa4
+	github.com/temporalio/tctl v0.0.0-20210730234214-fa5d45a3ed6c
```

Comment:
This is causing circular dependency problem. The tctl depends on server, and here server depends on tctl.